apps/guide/src/content/04-popular-topics/03-formatters.mdx

```diff
@@ -0,0 +1,95 @@
+---
+title: Formatters
+category: Popular topics
+---
+
+# Formatters
+
+discord.js provides the <DocsLink package="formatters" /> package which contains a variety of utilities you can use when writing your Discord bot.
+
+## Basic Markdown
+
+These functions format strings into all the different markdown styles supported by Discord.
+
+<CH.Code>
+
+```js
+import { bold, italic, strikethrough, underscore, spoiler, quote, blockQuote } from 'discord.js';
```

Comment:
Shouldn't this show imports from `@discordjs/formatters` then, since its a guide for it? 👀 Or should it instead be mentioned as an alternative?